bodhi/server/services/graphql.py

```diff
@@ -17,18 +17,56 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 """Defines API endpoints related to GraphQL objects."""
 from cornice import Service
+from webob_graphql import serve_graphql_request
+import graphene
+
+from bodhi.server.graphql_schemas import Release
+
 
 graphql = Service(name='graphql', path='/graphql', description='graphql service')
 
 
 @graphql.get()
 def graphql_get(request):
     """
-    Return "Hello World".
+    Perform a GET request.
+
+    Args:
+        request (pyramid.Request): The current request.
+    Returns:
+        The response from Bodhi to the request.
+    """
+    context = {'session': request.session}
+
+    # Setting graphiql_enabled=True
+    return serve_graphql_request(request, schema, graphiql_enabled=True, context_value=context)
```

Comment:
I think we want to have the `graphiql_enabled` value to be configure in the configuration so that we can have it set to `True` in the development environment and `False` in production.

Reply:
For adding this in the configuration you can look at an example here --> 
import the configuration : 
https://github.com/fedora-infra/bodhi/blob/develop/bodhi/server/models.py#L47
get the value of a configuration key: 
https://github.com/fedora-infra/bodhi/blob/develop/bodhi/server/models.py#L1094
add the configuration key to the file : 
https://github.com/fedora-infra/bodhi/blob/develop/production.ini#L322
and https://github.com/fedora-infra/bodhi/blob/develop/devel/development.ini.example

Reply:
the configuration key could be something like `enable_graphiql`